basil/HL/mercury.py

```diff
@@ -56,8 +73,42 @@ def get_channel(self, address=None):
         self._write_command("TS", address)
         return self.read()
 
-    def set_position(self, value, address=None):
+    def set_position(self, value, precision=500, address=None, wait=True):
         self._write_command("MA%d" % value, address)
+        if wait is True:
+            pos = self._wait_FE(address)
+            if abs(pos - value) <= precision:
+                print("At position", pos, "Traget at", value)
+            else:
+                print("Warning: Precision not reached")
```

Comment:
Should be something like: "Target not reached", no? Maybe one could print the target, actual value, and prec, too.